app/assets/stylesheets/application.css

```diff
@@ -77,3 +78,60 @@ th {
    color: #468847 ;
    text-align: left;
 }
+
+.simple-calendar {
+  width: 1024px;
+  height: 600px;
+}
+
+  .day {
+    height: 140px;
+    vertical-align: top;
+  }
+
+  .wday-0 {
+    border: 3px solid #FC8282;
+    min-width:150px;
+  }
+  .wday-1 {
+    min-width:140px;;
+  }
+  .wday-2 {
+    min-width:140px;
+  }
+  .wday-3 {
+    min-width:140px;
+  }
+  .wday-4 {
+    min-width:140px;
+  }
+  .wday-5 {
+    min-width:140px;
+  }
+  .wday-6 {
+    min-width:140px;
+  }
```

Comment:
All of these CSS classes are the same. Do we need these or do you expect them to stay the same?